parts/scenes/dict.lua

```diff
@@ -59,10 +59,13 @@ end
 local function _search()
     local input=inputBox:getText():lower()
     _clearResult()
+    -- [SEA]: Can the search function prioritize the best match? Yes for almost all the time except Vietnamese
```

Comment:
> you can just remove `[SEA]:` from the comment, there is no need to do that you don't see MrZ making comments like `-- Z: insert comment here`, so you should consider following that

Do you think it's okay because that changes is made by me, not by MrZ
I'm afraid someone will mistake me for MrZ who made this change